src/recurrenceIterator.test.ts

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable no-mixed-operators */
-import moment from 'moment';
-import 'moment-timezone';
+import * as moment from 'moment-timezone/builds/moment-timezone-with-data-2012-2022';
```

Comment:
```ts
import * as moment from 'moment'
import 'moment-timezone'
```
is the same as 
```ts
import * as moment from 'moment-timezone'
```
so we can import the build with data directly (for timezone support in browser bundles)
```ts
import * as moment from 'moment-timezone/builds/moment-timezone-with-data-2012-2022'
```